laravel-deploy/app-release

```diff
@@ -196,10 +196,22 @@ sudo chmod 777 "$CURRENT/storage" -R
 sudo chmod 777 "$CURRENT/bootstrap" -R
 
 ## Make Logs Directory
-sudo mkdir -p "$REPO_ROOT/$LARAVELLOGSFOLDER_LOCATION"
+sudo mkdir -m 775 -p "$REPO_ROOT/$LARAVELLOGSFOLDER_LOCATION"
```

Comment:
this will make the end directory have the correct permissions, but not the full tree. hence the looping afterward